core/admin/mailu/models.py

```diff
@@ -255,20 +255,23 @@ def dns_autoconfig(self):
         """ return list of auto configuration records (RFC6186) """
         hostname = app.config['HOSTNAME']
         protocols = [
-            ('submission', 587),
-            ('imap', 143),
-            ('pop3', 110),
+            ('imap', 143, 20),
+            ('pop3', 110, 20),
+            ('submission', 587, 20),
         ]
         if app.config['TLS_FLAVOR'] != 'notls':
             protocols.extend([
-                ('imaps', 993),
-                ('pop3s', 995),
+                ('autodiscover', 443, 10),
+                ('submissions', 465, 10),
+                ('imaps', 993, 10),
+                ('pop3s', 995, 10),
             ])
+
         return list([
```

Comment:
The list() is not necessary anymore when adding (appending) a list.

Reply:
That's 3aa735cc